pkg/mount/mountutil_test.go

```diff
@@ -64,40 +69,87 @@ func TestGetMountAttr(t *testing.T) {
 
 	mountAttrTests := map[string]struct {
 		devPath           string
+		mountPoint        string
+		attrFunc          int
```

Comment:
Why is this required? You can use the type [`getMountData`](https://github.com/openebs/node-disk-manager/blob/master/pkg/mount/mountutil.go#L44)

Reply:
This is because the functions `getMountName` and `getPartitionName` are methods of `DiskMountUtil`. I can't reference them before creating the mountutil which is being created inside the test.

Reply:
ok. got it.